api/src/play_asset_pack_manager.gd

```diff
@@ -82,6 +94,25 @@ func _initialize_plugin() -> Object:
 		push_error("Android plugin singleton not found!")
 		return null
 
+# -----------------------------------------------------------------------------
+# Helper functions that receive signals emitted from the plugin
+# -----------------------------------------------------------------------------
+func show_cellular_data_confirmation_success(result : int, signal_id : int):
```

Comment:
Good catch, I added underscore prefix for these functions.

Reply:
I also added underscore for request callback functions such as ```_on_show_cellular_data_confirmation_success``` since developers should not be calling them.